pydundas/rest/notification.py

```diff
@@ -14,7 +15,7 @@ def __init__(self, session=None, api=None):
 
     def getByName(self, name):
         """Get all notification with this exact name. Case insensitive."""
```

Comment:
Not part of the changes, missed by earlier speling nasi review: `s/notification/notifications/`.